### ISSUE TITLE
Forward button appearance does not match back button

### ISSUE THREAD

**Issue by BlitW0** [coekuss/quietfox#15]
The forward button looks like the other toolbar button (for e.x., no circular border around it), but should look like the back button. I tried adding `#forward-button` rules everywhere `#back-button` appeared, mimicking existing rules for `#back-button`. Still the forward button shows up like other toolbar buttons

**Comment by yokoffing** [comment]
I go in and tell Firefox to use the compact density for the address bar and tabs. Compact density always removes the circle around the back button no matter what your setup looks like.

![2021-01-31 (2)](https://user-images.githubusercontent.com/11689349/106390489-b5c95380-63b6-11eb-8b7d-762294435bc3.png)

(I'm not the maintainer. I don't know if he originally designed Quietfox to remove the circle around the backspace button in Normal density.)

**Comment by BlitW0** [comment]
Yeah, I tried that already and it worked. I guess that's just a shortcoming of firefox itself.

**Comment by yokoffing** [comment]
> Yeah, I tried that already and it worked. I guess that's just a shortcoming of firefox itself.

Idk if "shortcoming" is an appropriate label. It's a design choice to have a circle around the back button in normal and touch density.

**Comment by BlitW0** [comment]
Well, I only meant it in the sense that it's inconsistent to have a circle around the back button and not on the forward button.
